lib/HLSL/DxilValidation.cpp

```diff
@@ -2345,7 +2345,83 @@ static void ValidateDxilOperationCallInProfile(CallInst *CI,
           CI, ValidationRule::SmOpcodeInInvalidFunction,
           {"CreateHandle", "Shader model 6.5 and below"});
     }
+  } break;
+    
+  // this intrinsic is generated when the SV_DispatchThreadID system value is used
+  case DXIL::OpCode::ThreadId: {  
```

Comment:
Using a minimal comment (like `// SV_DispatchThreadID`) at the end of the case statement line would probably suffice for each of these.